File: 2015/pratica/collatz/att/collatz.pas
function congettura(N: longint): longint;
begin
    (* Mettete qui il codice della soluzione *)
    congettura := 42;
end;


var
    N: longint;
    fr, fw: text;
begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N);

    writeln(fw, congettura(N));
    close(fr);
    close(fw);
end.

File: ed1/pratica/collatz/att/collatz.c
#include <stdio.h>
#include <assert.h>

int congettura(int N) {
    // Mettete qui il codice della soluzione
    return 42;
}


int main() {
    FILE *fr, *fw;
    int N;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");

    assert(1 == fscanf(fr, "%d", &N));
    
    fprintf(fw, "%d\n", congettura(N));
    fclose(fr);
    fclose(fw);
    return 0;
}
